.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+* **Kind:** bugfix / enhancement / feature
+
+<!-- For a bug fix, make sure the bug was already reported in an issue. -->
+
+* Close #…
+
+### Description
+
+<!-- Please recap the motivation and/or implementation choices for this change. -->
+
+### Checklist
+
+<!-- You can remove all the check-boxes that are not applicable. -->
+
```

Comment:
We could add a specific item for the format of commit messages as it is really important for our release process.

Reply:
Sorry, the links were broken in my previous message (because #452 is not yet merged).
See instead:
https://github.com/ocaml-sf/learn-ocaml/blob/d3c7e9b093eeed1951137060b66c59d4711055c6/.github/PULL_REQUEST_TEMPLATE.md#checklist